apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SpannerClient.Streaming.cs

```diff
@@ -28,7 +28,7 @@ public partial class SpannerClient
         /// <param name="request"></param>
         /// <param name="callSettings"></param>
         /// <returns></returns>
-        internal virtual AsyncServerStreamingCall<PartialResultSet> ExecuteSqlStream(ExecuteSqlRequest request, CallSettings callSettings = null)
+        public virtual AsyncServerStreamingCall<PartialResultSet> ExecuteSqlStream(ExecuteSqlRequest request, CallSettings callSettings = null)
```

Comment:
Eek, hadn't spotted that this was generated.
Will backtrack to work out why I made these public.
(Not sure which partial class you're asking about in terms of modifying it - we really don't want to modify anything autogenerated though.)